src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +98,10 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates.stream()
+                .filter(i -> new CandidateValidator().test(i))
```

Comment:
no need to call a test method, check the type that `filter()` accepts
```suggestion
                .filter(new CandidateValidator())
```